package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "lovefield",
   "version": "2.1.12",
+  "types": "dist/lovefield.d.ts",
```

Comment:
Changes in this file are unrelated to the functionality mentioned (it just fixes some build issues, type issues & npm audit complains). Since this PR isn't getting merged I just put all my changes into a single PR